### DISCUSSION TITLE
Swipe borders

### DISCUSSION THREAD

**Discussion by 14837-eng** [nolimits4web/swiper#5226] [Q&A] [answered]
How can you limit the swipe when the first or last slide is active, so as not to go beyond the boundaries?

![image](https://user-images.githubusercontent.com/56313646/143374228-9b1dd7e2-5232-4f1b-9100-f321d7c8217e.png)

**Comment by nolimits4web** [comment, marked as answer]
Set `resistanceRatio: 0` parameter

**Reply by 14837-eng** [reply]
thnx
